lua/luasnip/init.lua

```diff
@@ -39,6 +42,9 @@ local function match_snippet(line, type)
 	)
 end
 
+---@alias GetSnippets fun(ft: string|nil, opts: table|nil): table
```

Comment:
```suggestion
---@alias GetSnippets fun(ft: string|nil, opts: table|nil): table<string, Snippet[]>
```

Just a tad more information: the table maps filetypes to Snippets, like `{"all" = {snip1, snip2, ...}, "lua" = {snip3, ...}, ...}`, maybe it helps :D